packages/coinstac-client-core/src/sub-api/auth.js

```diff
@@ -35,27 +33,23 @@ class Auth {
    * Makes a request to the API to create a new user. Response should contain
    * new user info.
    *
-   * @param {Object}  data          Information to create a new user
-   * @param {string}  data.username
-   * @param {string}  data.email
-   * @param {string}  data.name     New user's first name and last name
-   * @param {string}  data.password
+   * @todo Use user's site ID
+   *
+   * @param {Object} data Information to create a new user
+   * @param {string} data.email
+   * @param {string} data.name New user's first name and last name
+   * @param {string} data.password
+   * @param {string} data.username
    * @returns {Promise}
    */
-  createUser(user) {
-    // the API requires a `label` key instead of `name`
-    /* istanbul ignore next */
-    const newUser = {
-      email: user.email,
-      label: user.name,
-      password: user.password,
-      username: user.username,
-      siteId: '7', // @TODO :(
-    };
-
-    // @TODO use actual user siteId
-    /* istanbul ignore next */
-    return bluebird.resolve(this.halfpenny.users.post(newUser));
+  createUser({ email, name, password, username }) {
+    return this.halfpenny.users.post({
+      email,
+      label: name,  // the API requires a `label` key instead of `name`
+      password,
+      siteId: '7',
+      username,
```

Comment:
username and pass need to be base64 :-) https://github.com/MRN-Code/quarterback/blob/master/packages/steelpenny/src/lib/controllers/users.js#L144

This is newish for the PR site